app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogView.kt

```diff
@@ -162,14 +191,40 @@ class AppRatingDialogView(context: Context) : LinearLayout(context), OnRatingBar
     private fun setup(context: Context) {
         LayoutInflater.from(context).inflate(R.layout.component_app_rate_dialog, this, true)
         ratingBar = findViewById(R.id.app_rate_dialog_rating_bar) as CustomRatingBar
-        editText = findViewById(R.id.app_rate_dialog_edit_text) as EditText
+        commentEditText = findViewById(R.id.app_rate_dialog_comment_edit_text) as EditText
         titleText = findViewById(R.id.app_rate_dialog_title_text) as TextView
-        contentText = findViewById(R.id.app_rate_dialog_content_text) as TextView
+        descriptionText = findViewById(R.id.app_rate_dialog_description_text) as TextView
         noteDescriptionText = findViewById(R.id.app_rate_dialog_note_description) as TextView
         ratingBar.setIsIndicator(false)
         ratingBar.setOnRatingBarChangeListener(this)
     }
 
+    fun setCursorColor(view: EditText, @ColorInt color: Int) {
```

Comment:
This seems a bit hacky. Maybe this could work instead https://stackoverflow.com/a/9165217/973379 ?